### DISCUSSION TITLE
Cannot subscribe to issue state change (closed/re-open) anymore with the new issues UI ☹

### DISCUSSION THREAD

**Discussion by bew** [community/community#141198] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

In the new (opt-in?) UI for issues, we can only subscribe to all or unsubscribe from all ([example issue](https://github.com/neovim/neovim/issues/20451)):
![image](https://github.com/user-attachments/assets/6bc40d8d-342a-442c-b2bf-a42a90505f52)

---

Whereas for the previous UI had the ability to customize the subscription notification ([example issue](https://github.com/jake-stewart/multicursor.nvim/issues/40)):
![image](https://github.com/user-attachments/assets/647825b7-eac0-413a-a1c3-e883fe91e94c)
And clicking `Customize` gives this menu:
![image](https://github.com/user-attachments/assets/61cbc119-f9ea-4bbd-98b8-9cdebc8ef85f)

----
👉 This is especially annoying as I like to subscribe to MANY projects and subscribe to some issues' closed/re-open events to know something was impl/cancelled and be able to check the eventual discussion afterward..

❔ Anyone know if it is only temporary? (I hope so)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by segln** [comment]
This is really annoying :(

**Comment by glen-84** [comment]
https://github.com/orgs/community/discussions/132506#discussioncomment-10122125
https://github.com/orgs/community/discussions/139935#discussioncomment-10830138

**Reply by segln** [reply]
Thanks for the information! I will use Refined Github.

**Comment by azenMatt** [comment]
We are working on adding this back :)
